sql/src/main/java/io/crate/operation/udf/UDFLanguageRegistry.java

```diff
@@ -0,0 +1,23 @@
+package io.crate.operation.udf;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UDFLanguageRegistry {
```

Comment:
the method of this class can be part of the `UserDefinedFunctionService.java` class
because it is only a wrapper around a map